src/helpers/crypto.cpp

```diff
@@ -1,24 +1,25 @@
 #include "helpers/crypto.h"
 
-#include "helpers/helpers.h"
-#include "helpers/encoding/der.h"
+#include <cassert>
 #include "bcl/Ecdsa.hpp"
 #include "bcl/Sha256.hpp"
 #include "bcl/Uint256.hpp"
+#include "helpers/encoding/der.h"
+#include "helpers/helpers.h"
 #include "rfc6979/rfc6979.h"
 #include "uECC.h"
-#include <cassert>
-
 
-void cryptoSign(Sha256Hash hash, Ark::Crypto::Identities::PrivateKey privateKey, std::vector<uint8_t>& signature)
-{
+void cryptoSign(Sha256Hash hash, Ark::Crypto::Identities::PrivateKey privateKey,
+                std::vector<uint8_t>& signature) {
   Uint256 r;
   Uint256 s;
 
   uint8_t nonce32[32] = {};
-  nonce_function_rfc6979(nonce32, hash.value, privateKey.toBytes(), nullptr, nullptr, 0);
+  nonce_function_rfc6979(nonce32, hash.value, privateKey.toBytes(), nullptr,
```

Comment:
Ones like these can't be accurate. Doesn't seem like only one parameter should be put on a new line.